cmd/build.go

```diff
@@ -168,13 +168,15 @@ var (
 	commandList     bool
 	commandDb       bool
 	dependencyGraph bool
+	numThreads      int
 )
 
 func init() {
 	rootCmd.AddCommand(buildCmd)
 	buildCmd.Flags().BoolVar(&commandList, "commands", false, "Create compile commands list")
 	buildCmd.Flags().BoolVar(&commandDb, "compdb", false, "Create compile commands JSON database")
 	buildCmd.Flags().BoolVar(&dependencyGraph, "graph", false, "Create dependency graph")
+	buildCmd.Flags().IntVarP(&numThreads, "threads","j", -1, "Run N jobs in parallel")
```

Comment:
maybe make this a `uint` with 0 being the default? that way we don't have the weird `-j0` case that would forward to ninja